src/main/java/com/epam/ms/user/UserProfileHandler.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.ms.user;
```

Comment:
Let's try one more tool: Actuator. It's widely used in production
[Link](https://www.baeldung.com/spring-boot-actuators)

After that read [this](https://www.baeldung.com/spring-reloading-properties#reloading-cloud)
And try to play with it. Start app, try to call it. Find the results. Then go to your property files. Change values. Call /refresh endpoint. Call service one more time. Find new results.